doc/api.md

```diff
@@ -60,89 +45,68 @@ Reference table for genes within Cognoma. Entire model is read-only.
 | standard_name | string |  |
 | description | string |  |
 | organism | integer | Foreign Key referencing the Organism. |
-| aliases | string |  |
+| aliases | string | space-separated list of aliases |
 | obsolete | boolean |  |
-| weight | boolean |  |
-| sample_mutation_sumary | object | Count/aggregates of sample mutations available in Cognoma for this particular gene. |
-
-#### Tissues (/tissues)
-
-Entire model is read-only.
-
-| Field        | Type           | Description |
-| ------------- |:-------------:| ----------:|
-| acronym | string | Primary Key. Ex "GBM" |
-| name | string | Full tissue name. Ex "Glioblastoma Multiforme" |
+| weight | boolean | Weight used for search results since multiple genes may have the same symbol |
 
 ### Organism (/organisms)
 
-Reference table for organisms within Cognoma. Entire model is read-only.
-
+Reference table for organisms within Cognoma. Entire model is read-only. This table is created using [`django-organisms`](https://bitbucket.org/greenelab/django-organisms). Note that Project Cognoma will deal exclusively with human genes (when taxonomy_id equals 9606).
 
 | Field        | Type           | Description |
 | ------------- |:-------------:| ----------:|
 | taxonomy_id | integer | Primary Key. Taxonomy ID assigned by NCBI. |
 | common_name | string | Organism common name, e.g. 'Human' |
 | scientific_name | string | Organism scientific/binomial name, e.g. 'Homo sapiens' |
 
-### Sample Mutation Summary (embedded in Gene)
+### Disease Types (/diseases)
 
-Count/aggregates of sample mutations available in Cognoma for a particular gene. Entire model is read-only.
+Entire model is read-only.
 
 | Field        | Type           | Description |
 | ------------- |:-------------:| ----------:|
-| count | integer | Number of mutations. |
+| acronym | string | Primary Key. Ex "GBM" |
+| name | string | Full tissue name. Ex "Glioblastoma Multiforme" |
 
 ### Sample (/samples)
 
 Select data from the TCGA database used for reference within Cognoma. Entire model is read-only.
 
-
-| Field        | Type           | Description |
-| ------------- |:-------------:| ----------:|
-| tcga_id | string | Primary Key. Sample ID assigned by TCGA. |
-| mutations | object | Gene mutation data. |
+| Field        | Type          | Description |
+| ------------ |:-------------:| ----------:|
+| sample_id | string | Primary Key. Sample ID assigned by TCGA. |
+| disease | string | Foreign key referencing the disease |
+| gender | string | male or female |
+| age_diagnosed | integer | patient age when cancer was diagnosed |
 
 ### Mutation (embedded in Sample)
 
 Sample to gene mutations.
 
 | Field        | Type           | Description |
 | ------------- |:-------------:| ----------:|
-| mutation_type | string |  |
-| gene | object | Gene associated with this mutation. |
+| sample_id | string | Foreign Key referencing samples |
+| entrez_gene_id | object | Foreign Key referencing a mutated gene for the sample |
```

Comment:
Do we still want to keep the gene as an optional join? We could also wait on the frontend for this. What that would mean is that you wouldn't have to pull all the joins and lookup the gene id to do things like display the name of the gene